modules/customeradd.php

```diff
@@ -93,6 +93,11 @@
 	if($customeradd['name'] == '' && !$customeradd['type'])
 		$error['name'] = trans('First name cannot be empty!');
 	
+	if (ConfigHelper::getConfig('phpui.add_customer_group_required', false)) {
```

Comment:
Zgodnie z tym co jest w metodzie checkValue jest to jak najbardziej prawidłowe zachowanie.
opakowane.